common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -179,6 +179,18 @@ public ManagedBuffer getBlockData(
     return getSortBasedShuffleBlockData(executor, shuffleId, mapId, reduceId);
   }
 
+  public ManagedBuffer getBlockData(
```

Comment:
I think it might be better to call this `getRddBlockData` just to make clear that it's not getting shuffle data. The argument name (`rddId`) is too easy to overlook.